core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/scatter3D/Scatter3dChartOpDesc.scala

```diff
@@ -0,0 +1,122 @@
+package edu.uci.ics.texera.workflow.operators.visualization.scatter3D
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaTitle}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+@JsonSchemaInject(json = """
+{
+  "attributeTypeRules": {
+    "title": "string"
+  }
+}
+""")
+class Scatter3dChartOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+  @JsonProperty(value = "title", required = true)
+  @JsonSchemaTitle("Title")
+  @JsonPropertyDescription("chart title")
+  var title: String = ""
+
+  @JsonProperty(value = "x", required = true)
+  @JsonSchemaTitle("Value X")
+  @JsonPropertyDescription("the value x axis")
+  @AutofillAttributeName
+  var x: String = ""
+
+  @JsonProperty(value = "y", required = true)
+  @JsonSchemaTitle("Value Y")
+  @JsonPropertyDescription("the value y axis")
+  @AutofillAttributeName
+  var y: String = ""
+
+  @JsonProperty(value = "z", required = true)
+  @JsonSchemaTitle("Value Z")
+  @JsonPropertyDescription("the value z axis")
+  @AutofillAttributeName
+  var z: String = ""
+
+  override def getOutputSchema(schemas: Array[Schema]): Schema = {
+    Schema.newBuilder.add(new Attribute("html-content", AttributeType.STRING)).build
+  }
+
+  override def operatorInfo: OperatorInfo =
+    OperatorInfo(
+      "Scatter3D",
```

Comment:
Rename to `Scatter3D Plot` or `Scatter3D Chart`.

Reply:
Fixed.